worker.go

```diff
@@ -70,35 +70,35 @@ func (w *Worker) ProcessMetric(m *Metric) {
 			log.WithField("name", m.Name).Debug("New counter")
 			w.counters[m.Digest] = NewCounter(m.Name, m.Tags)
 		}
-		w.counters[m.Digest].Sample(m.Value, m.SampleRate)
+		w.counters[m.Digest].Sample(m.Value.(float64), m.SampleRate)
```

Comment:
yeah for now unchecked conversions should work without panic since they're tied to the metric.Type. Those are words I'll probably end up eating, but atm I really don't want to introduce error returns to this part of the callstack. The alternative, differentiating the metrics by type, requires a lot of extra type cruft. Maybe a false dichotomy? We can come back to this when we think the implementation demands it